base/linalg/triangular.jl

```diff
@@ -1888,10 +1898,7 @@ function sqrtm{T}(A::UpperTriangular{T})
     for j = 1:n
```

Comment:
You can just put the `@inbounds` in front of the `for ... end` group no need of the extra `begin ... end` here. That is,

```julia
@inbounds for j = 1:n
    # loop body
end
```